client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -195,10 +184,49 @@ class CalypsoifyIframe extends Component<
 		}
 	}
 
+	componentDidUpdate( { shouldLoadIframe }: ComponentProps ) {
+		// If shouldLoadIframe changed from false to true, and the timer hasn't yet been set.
+		if ( ! this.editorRedirectTimer && ! shouldLoadIframe && this.props.shouldLoadIframe ) {
+			this.setEditorRedirectTimer( 2500 );
```

Comment:
Is the `2.5s` on purpose, or was it meant to be `25s`?

Reply:
thanks for catching this!